From: xsmile <>
Date: Thu, 11 Oct 2018 21:29:35 +0200
Subject: Allow sign-in to Google sites like Gmail without signing in to
 Chromium

Disable prefs::kSigninAllowedOnNextStartup by default. The setting can be found in chrome://settings

---
 chrome/browser/signin/account_consistency_mode_manager.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/chrome/browser/signin/account_consistency_mode_manager.cc b/chrome/browser/signin/account_consistency_mode_manager.cc
--- a/chrome/browser/signin/account_consistency_mode_manager.cc
+++ b/chrome/browser/signin/account_consistency_mode_manager.cc
@@ -172,7 +172,7 @@ void AccountConsistencyModeManager::RegisterProfilePrefs(
   registry->RegisterBooleanPref(prefs::kAccountConsistencyMirrorRequired,
                                 false);
 #endif
-  registry->RegisterBooleanPref(prefs::kSigninAllowedOnNextStartup, true);
+  registry->RegisterBooleanPref(prefs::kSigninAllowedOnNextStartup, false);
 }
 
 // static
-- 
2.11.0

